atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -36,6 +39,18 @@ private Object doDelete(RegeneratingTable<Multimap<Cell, Long>> table) {
         return table.getTableCells();
     }
 
+    private Object doDeleteRange(ConsecutiveNarrowTable table, int numBatches) {
+        Iterable<RangeRequest> rangeRequests;
+        if (numBatches == 1) {
+            rangeRequests = ImmutableList.of(RangeRequest.all());
+        } else {
+            rangeRequests = table.getRangeRequests(1, table.getNumRows() / numBatches);
```

Comment:
Beware: the time spent getting the ranges will count when you're running the benchmark. Do you want this?

I think there's a way to avoid it (or you could probably just hard-code the ranges) - as he's co-located, @rjullman might be the best person to help you out.